README.md

```diff
@@ -44,12 +43,31 @@ $ cd cozy-drive
 $ yarn install
 ```
 
-:pushpin: If you use a node environment wrapper like [nvm] or [ndenv], don't forget to set your local node version `8` before doing a `yarn install`.
+:pushpin: Don't forget to set your local node version `12` before doing a `yarn install`.
 
-:warning: During its early ages, _cozy-drive_ uses beta versions of [cozy-ui] and [cozy-client-js], take a look at the ["living on the edge" note](#living-on-the-edge) below to know hot to install and configure the latest available versions.
+:warning: During its early ages, _cozy-drive_ uses beta versions of [cozy-ui] and [cozy-client-js], take a look at the ["living on the edge" note](#living-on-the-edge) below to know how to install and configure the latest available versions.
 
 Cozy's apps use a standard set of _npm scripts_ to run common tasks, like watch, lint, test, build…
 
+### Run in dev mode
+
+Using a watcher:
+```sh
+$ cd cozy-drive
+$ yarn watch:drive:browser
+$ cozy-stack serve --appdir drive:/<project_absolute_path>/cozy-drive/build/drive
+```
+
+Or directly Build the app and run in cozy-stack:
+```sh
+$ cd cozy-drive
+$ yarn build:drive
+$ cozy-stack serve --appdir drive:/<project_absolute_path>/cozy-drive/build/drive
+```
+
+Your app is available at http://drive.cozy.localhost:8080/#/folder
+
+Note: it's mandatory to explicit to cozy-stack the folder of the build that should be served, for Hot module replacement.
```

Comment:
This is not for HMR. This is "to be able to run the app". If you run watch (hmr enabled) or build (static file generated) you need to tell the stack where to serve the app from.

Reply:
updated